src/shared/cloudformation/templateSymbolResolver.ts

```diff
@@ -113,19 +114,13 @@ export class TemplateSymbolResolver {
 
 export class TemplateSymbolProvider {
     public async getSymbols(document: vscode.TextDocument, waitForSymbols: boolean): Promise<vscode.DocumentSymbol[]> {
```

Comment:
`waitForSymbols` is now unused. Do we want to add a contingency for that in `waitUntil`?